src/axolotl/core/trainer_builder.py

```diff
@@ -836,6 +836,13 @@ def get_callbacks(self) -> List[TrainerCallback]:
                 SaveAxolotlConfigtoWandBCallback(self.cfg.axolotl_config_path)
             )
 
+        if self.cfg.use_vessl:
+            from axolotl.utils.callbacks.vessl_ import VesslLogMetricsCallback
```

Comment:
could you tell me why you added a suffix `_` to `vessl_`?

Reply:
They added underscore too on mlflow (`mlflow_`) and wandb (`wandb_`) package, so I think it's the convention for external integration in this repo.